projects/pokemon_game.py

```diff
@@ -28,41 +29,36 @@ def __init__(self, name, primary_type, max_hp, hp) -> None:
     def battle (self, computer_pokemon):
         # water > fire > grass > water
         if self.primary_type == computer_pokemon.primary_type:
-            print(f"Both Pokemon are type {self.primary_type}, it's a tie! They both loose 1 HP. \n")
-            user_pokemon.hp -= 1
-            computer_pokemon.hp -= 1
+            print(f"Both Pokemon are type {self.primary_type}, it's a tie! You both loose 5 HP. \n")
+            user_pokemon.hp -= 5
```

Comment:
**suggestion**: Right now, you have the `.battle()` method that directly sets the `hp` of both pokemon and also performs the checks on the pokemon to see if they are at max HP or have died.

I suggest you create an extra method for your pokemon that can be called something like `.change_hp()` that can have one parameter for `change`. So you'd call it like this:

```py
user_pokemon.change_hp(-5)
```

Then within the `.change_hp()` method, you'd check if the pokemon has died or has max_hp. This might mean that your `feed()` mehod becomes a very thin wrapper, so you might not need it in the end.

This would do a lot to make the large nested  `if` block a lot easier to absorb. Large nested `if` blocks generally should be avoided if possible.